backend/test/acceptance/api.terminals.spec.js

```diff
@@ -168,16 +214,18 @@ module.exports = function info ({ agent, sandbox, k8s, auth }) {
       })
 
       const res = await agent
-        .post(`/api/namespaces/${namespace}/terminals/${target}`)
+        .post('/api/terminals')
         .set('cookie', await user.cookie)
         .send({
           method: 'fetch',
           params: {
-            namespace,
-            name
+            coordinate: {
```

Comment:
I am confused. Now name and namespace are outside of coordinate. To me still I don't understand what a coordinate should be. We need to find a name which better describes what is and what it is used for.